component-samples/charts-components/samples/vue/vite.config.js

```diff
@@ -1,29 +1,26 @@
-/* Copyright 2024 Esri
- *
- * Licensed under the Apache License Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
+import { fileURLToPath, URL } from 'node:url'
 
-import { defineConfig } from 'vite';
-import vue from '@vitejs/plugin-vue';
+import { defineConfig } from 'vite'
+import vue from '@vitejs/plugin-vue'
```

Comment:
revert?

Reply:
map-components [Vue sample](https://github.com/Esri/jsapi-resources/blob/93d462461c82c6c98e1461d55e402b79ba8306d7/component-samples/map-components/samples/vue/vite.config.js#L3-L4) also does not have the semicolon after imports, but coding components [Vue sample](https://github.com/Esri/jsapi-resources/blob/93d462461c82c6c98e1461d55e402b79ba8306d7/component-samples/coding-components/samples/vue/vite.config.js#L2-L3) does. Which pattern should we follow? 

@omarkawach

Reply:
@jimmychen97 might as well add semicolons for consistency with the rest of the samples in the repo.